ToggleButton/ToggleButton.less

```diff
@@ -61,6 +62,17 @@
 					border-color: @agate-toggle-button-switch-on-focus-border-color;
 				}
 			}
+
+
+			.client {
+				.selected& {
+					color: @agate-toggle-focus-text-color;
+
+					.underline& {
```

Comment:
The `&` here may not be what you want here. Verify the selector that is generated from this since the `&` copies all previous selectors in the hierarchy when it's used. (Nesting `&` may generate too many unwanted selectors.)